packages/test-utils/src/reporter.ts

```diff
@@ -977,116 +992,124 @@ class ReplayReporter<TRecordingMetadata extends UnstructuredMetadata = Unstructu
   }
 
   async onEnd(): Promise<PendingWork[]> {
-    debug("onEnd");
-
-    const output: string[] = [];
-    let completedWork: PromiseSettledResult<PendingWork>[] = [];
-
-    if (this._pendingWork.length) {
-      log("🕑 Completing some outstanding work ...");
-    }
+    try {
+      debug("onEnd");
 
-    while (this._pendingWork.length) {
-      const pendingWork = this._pendingWork;
-      debug("Outstanding tasks: %d", pendingWork.length);
+      await this._cacheAuthIdsPromise;
 
-      this._pendingWork = [];
-      completedWork.push(...(await Promise.allSettled(pendingWork)));
-    }
+      this._logger.info("Identification added to logger instance");
```

Comment:
Should this log be moved up inside of the `getAuthIds.then`? Where we actually call `_logger.identify`? Else it may be pretty decoupled from where we _actually_ identify the user/session.

(Similar logic for why the "_failed to add auth ids to the logger_" error log is up there in the `.catch`)

Reply:
I agree!